general/OpenMP.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * \version     v0.8
+ * \copyright   <2009-2019> Forschungszentrum J?lich GmbH. All rights reserved.
+ *
+ * \section License
+ * This file is part of JuPedSim.
+ *
+ * JuPedSim is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * JuPedSim is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with JuPedSim. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * \section Description
+ * The Simulation class represents a simulation of pedestrians
+ * based on a certain model in a specific scenario. A simulation is defined by
+ * various parameters and functions.
+ *
+ *
+ **/
+#pragma once
```

Comment:
what is the meaning of this line?

Reply:
Its a replacement for include guards:
```cpp
#indef FOO_H
#define FOO_H
... your code ...
#endif
```
can be replaced by a simple 
```cpp
#pragma once
```
it has 2 small benefits:
a) there can never be an name collision as with include guards. Imagine a situation where two include files use the same define as include guards. You could not include those headers in any file together.
b) single line vs 3 lines, less verbose

Reply:
that's interesting! Thanks!